client/src/pages/Feed.js

```diff
@@ -258,20 +261,57 @@ const Feed = () => {
     dispatchAction(TOGGLE_STATE, "showFilters");
   };
 
-  useEffect(() => {
+  const loadPosts = useCallback(async () => {
+    const limit = 5;
+    const skip = posts.page * limit;
     /* Add userId when user is logged */
-    const endpoint = "/api/posts"; // ?userId=xxxxxxxxx
-
-    postsDispatch({ type: FETCH_POSTS });
-    axios
-      .get(endpoint)
-      .then((response) => {
-        postsDispatch({ type: SET_POSTS, posts: response.data });
-      })
-      .catch((error) => {
-        postsDispatch({ type: ERROR_POSTS });
-      });
-  }, []);
+    const endpoint = `/api/posts?limit=${limit}&skip=${skip}`; // ?userId=xxxxxxxxx
+    let response = {};
```

Comment:
Why not store this in the initial state object?

Reply:
We only need response.data after a successful call.

Reply:
Ah I understand now